### PR TITLE
update getting_started.md to fix #23157

### LINKED ISSUE #23157: Getting Started guide should be more informative about :references
The getting started guide does not explain why it is generating a model with a :references data type column. It should be explained in a bit of details.. Section 6.1
http://guides.rubyonrails.org/getting_started.html

### PR DESCRIPTION
**Pull request by shakib609**
Updated getting started to provide a bit more information about :references data type when generating models.
Fix #23157

### CODE REVIEW

File: guides/source/getting_started.md [hunk 1]
```diff
@@ -1535,11 +1535,15 @@ class Comment < ApplicationRecord
   belongs_to :article
 end
 ```
-
 This is very similar to the `Article` model that you saw earlier. The difference
 is the line `belongs_to :article`, which sets up an Active Record _association_.
 You'll learn a little about associations in the next section of this guide.
 
+The (`:references`) keyword used in the bash command is a special data type for models. 
+It creates a new column on your database table with the provided name appended with an `_id`
```
Comment (maclover7) [inline guides/source/getting_started.md]: "with the provided name" --> "with the provided model name"

**Comment by maclover7** [comment]
One small grammar change, and then ready to merge!

**Comment by shakib609** [comment]
Fixed.. Let me know if something is wrong..

**Comment by vipulnsward** [comment]
LGTM. @shakib609 in future for [doc-only change](https://github.com/rails/rails/blob/master/guides/source/contributing_to_ruby_on_rails.md#contributing-to-the-rails-documentation), please commit with `[ci skip]` in the message, to avoid travis builds.

**Comment by shakib609** [comment]
Thanks for the suggestion @vipulnsward

**Comment by maclover7** [comment]
Please squash down, and include `[ci skip]` in the commit message to avoid creating a Travis build.

**Comment by shakib609** [comment]
Do you want me to make a new commit with `[ci skip]` ? @maclover7

**Comment by maclover7** [comment]
Yes, but please squash down to one commit.

**Pull request closed without merging**